### PR TITLE
[KFP] Introducing kfp 2 tech-preview support

### PR DESCRIPTION
**Pull request by liranbg**
Optional installation for kfp 2 alongside mlrun

This is a tech-preview and being used for our development effort into migration mlrun's kfp integration to kfp 2.

### CODE REVIEW

File: charts/mlrun-ce/templates/pipelines/configmaps/workflow-controller-configmap.yaml [hunk 1]
```diff
@@ -13,7 +13,9 @@ data:
       secretKeySecret:
         name: mlpipeline-minio-artifact
         key: secretkey
+  {{- if not (include "mlrun-ce.pipelinesV2" .) }}
   containerRuntimeExecutor: emissary
+  {{- end }}
```
Comment (TomerShor) [inline charts/mlrun-ce/templates/pipelines/configmaps/workflow-controller-configmap.yaml:18]: v2 doesn't need it?

File: charts/mlrun-ce/kfp2.yaml
Comment (TomerShor) [inline charts/mlrun-ce/kfp2.yaml:1]: Where / how is this being used?
Maybe document it somewhere?

File: charts/mlrun-ce/templates/pipelines/configmaps/workflow-controller-configmap.yaml (hunk 1 above)
Comment (liranbg) [reply charts/mlrun-ce/templates/pipelines/configmaps/workflow-controller-configmap.yaml:18]: v2 doesnt support it - as this is the only option and no other option is available

File: charts/mlrun-ce/kfp2.yaml
Comment (liranbg) [reply charts/mlrun-ce/kfp2.yaml:1]: will be added to mlrun org docs once it has better stability

**Review by TomerShor — APPROVED** [review]

**Pull request merged**